tests/sentry/incidents/endpoints/test_organization_alert_rule_details.py

```diff
@@ -537,7 +536,7 @@ def test_team_permission(self):
         ).delete()
         with self.feature("organizations:incidents"):
             resp = self.get_response(self.organization.slug, alert_rule.id, **serialized_alert_rule)
-        assert resp.status_code == 403
+        assert resp.status_code == 200
```

Comment:
These permissions tests change because we no longer block people from editing and deleting alerts based on team status.